raiden/transfer/state.py

```diff
@@ -414,7 +414,7 @@ class TokenNetworkState(State):
         'token_address',
         'network_graph',
         'channelidentifiers_to_channels',
-        'partneraddresses_to_channels',
+        'partneraddresses_to_channelidentifiers',
```

Comment:
This one has a lot more usage, I think it's fine to do this for now, but we should add an issue to remove this indirection in the future.